.circleci/create_circleci_config.py

```diff
@@ -285,7 +285,8 @@ def job_name(self):
         "pip install --upgrade --upgrade-strategy eager pip",
         "pip install -U --upgrade-strategy eager .[sklearn,tf-cpu,torch,testing,sentencepiece,torch-speech,vision]",
         "pip install -U --upgrade-strategy eager tensorflow_probability",
-        "pip install -U --upgrade-strategy eager -e git+https://github.com/huggingface/accelerate@main#egg=accelerate",
+        # Without this we can't dependency versions in the future
```

Comment:
d'oh sorry - bad original suggestion 

```suggestion
        # Without this we can't pin dependency versions in the future
```

Reply:
no worries, I forgot to get it in my expound too :)

Reply:
I guess you somehow forgot to change to `dependency versions` (unless that is what you intended to)